db-scripts/src/main/resources/migration.sql

```diff
@@ -380,4 +380,49 @@ ALTER TABLE `mutation` ADD COLUMN `DRIVER_FILTER_ANNOTATION` VARCHAR(80) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER` VARCHAR(50) NULL;
 ALTER TABLE `mutation` ADD COLUMN `DRIVER_TIERS_FILTER_ANNOTATION` VARCHAR(80) NULL;
 
-UPDATE info SET DB_SCHEMA_VERSION="2.4.0";
```

Comment:
@khzhu Please put `UPDATE info SET DB_SCHEMA_VERSION="2.4.0";` back in, which might be useful for future users who want to inspect 2.4.0, and to make it consistent with rest of migration.

Reply:
Hi @sandertan , sorry about that. I put update db schema version 2.4.0 sql statement back in and updated the PR.